Source/Machine.VSTestAdapter/Discovery/BuiltIn/SourceCodeLocationFinder.cs

```diff
@@ -90,6 +94,17 @@ private SourceCodeLocationInfo GetFieldLocationCore(TypeDefinition type, string
 
             return null;
         }
+
+        private bool IsHidden(SequencePoint sequencePoint)
+        {
+            const int lineNumberIndicatingHiddenLine = 16707566;
```

Comment:
Minor suggestion to encode this value as `0xFEEFEE` which slightly more google-ble  👍